modules/settings/assets/js/components/connect-modal/index.js

```diff
@@ -0,0 +1,48 @@
+import Box from '@elementor/ui/Box';
+import Button from '@elementor/ui/Button';
+import Grid from '@elementor/ui/Grid';
+import Modal from '@elementor/ui/Modal';
+import Typography from '@elementor/ui/Typography';
+import { __ } from '@wordpress/i18n';
+import { useAuth, useModal } from '../../hooks';
+
+function ConnectModal() {
+	const { isOpen } = useModal();
+	const { redirectToConnect } = useAuth();
+
+	return (
+		<Modal open={ isOpen }>
+			<Grid container
+				gap={ 2 }
+				direction="column"
+				justifyContent="start"
+				alignItems="center"
+				sx={ {
+					position: 'absolute',
+					top: '50%',
+					left: '50%',
+					transform: 'translate(-50%, -50%)',
+					width: 600,
```

Comment:
it will be broken on mobile

Reply:
@nirbhayel did you check this comment?

Reply:
@pkniazevych @DinetsV added support for mobile devices as well